cmd/miniooni/main.go

```diff
@@ -303,4 +306,11 @@ func main() {
 			}
 		}
 	}
+
+	if wrong {
+		log.Infof("...Finished with error! Duration : %v", time.Since(startTime))
+	}else {
+		log.Infof("...Finished perfectly! Thank you. Duration : %v", time.Since(startTime))
+	}
```

Comment:
I would strongly suggest to remove the code dealing with `startTime`. Both in the Unix shell and in the PowerShell there are mechanisms to measure the runtime of a command.

Reply:
OK. :D